src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -1340,6 +1341,100 @@ public bool WaitForExit(int milliseconds)
             return exited;
         }
 
+        /// <summary>
+        /// Instructs the Process component to wait for the associated process to exit, or
+        /// for the <paramref name="cancellationToken"/> to be canceled.
+        /// </summary>
+        /// <remarks>
+        /// Calling this method will set <see cref="EnableRaisingEvents"/> to true.
+        /// </remarks>
+        public async Task WaitForExitAsync(CancellationToken cancellationToken = default)
```

Comment:
The triple slash comments are missing a `<returns></returns>` item. Can you please add it too?

Reply:
Added!